packages/reference/src/assets/WrappedAssetCard/WrappedAssetCard.tsx

```diff
@@ -138,6 +139,18 @@ export const WrappedAssetCard = (props: WrappedAssetCardProps) => {
             }
           : undefined
       }
+      /* todo - remove this when onKeyDown is allowed as a prop for BaseCard in forma 36
+      // @ts-expect-error */
+      onKeyDown={
```

Comment:
❓ Don't know how it is handled, but when testing i found out that the WrappedAssetLink would also miss the onKeyDown handler, i would assume that it is out of this ticket?